src/main/java/org/carlmontrobotics/robotcode2023/Robot.java

```diff
@@ -26,6 +27,8 @@ public void robotInit() {
     DriverStation.startDataLog(DataLogManager.getLog());
     if(!DriverStation.isFMSAttached()) PathPlannerServer.startServer(5811);
     robotContainer = new RobotContainer();
+
+    SmartDashboard.putBoolean("safeMode", false);
```

Comment:
Oh wait is that what LiveWindow does with the `initSendable` method. Nvm what I said then

Reply:
One of my comments below talks about merging normal, slow, and baby mode into one variable instead of having two variables. You should probably change safeMode to operate similarly.